src/Components/Filter/Filters.js

```diff
@@ -71,20 +71,16 @@ const Filters = () => {
             <h3>Rating</h3>
             <ul className="list-items">
               {ratings.map((item) => (
-                <>
-                  <li key={item} className="list-item">
-                    <input
-                      type="radio"
-                      name={item}
-                      onChange={() =>
-                        dispatch({ type: "RATING", payload: item })
-                      }
-                      id={item}
-                      checked={rating === item}
-                    />
-                    <label htmlFor="#four-above"> {item} starts & above</label>
-                  </li>
-                </>
+                <li key={item} className="list-item">
+                  <input
+                    type="radio"
+                    name={item}
```

Comment:
Suggestion: Can use a little ambiguity here while naming, item can be a little confusing

Reply:
okay I'll do that